### PR TITLE
[OpAtttr]Add attribute var interface for Operator class

### PR DESCRIPTION
**Pull request by Aurelius84**
### PR types

Others
### PR changes

Others
### Describe

[OpAtttr]Add attribute var interface for Operator class.

### CODE REVIEW

File: python/paddle/fluid/framework.py [hunk 1]
```diff
@@ -3280,6 +3280,41 @@ def _blocks_attr_ids(self, name):
 
         return self.desc._blocks_attr_ids(name)
 
+    def _var_attr(self, name):
+        """
+        Get the Variable attribute  by name.
```
Comment (chenwhql) [inline python/paddle/fluid/framework.py:3285]: 好像多了个空格

File: python/paddle/fluid/framework.py [hunk 2]
```diff
@@ -3280,6 +3280,41 @@ def _blocks_attr_ids(self, name):
...
+        """
+        attr_type = self.desc.attr_type(name, True)
+        assert attr_type == core.AttrType.VAR, "Required type attr({}) is Variable, but received {}".format(
+            name, attr_type)
+        attr_var_name = self.desc.attr(name, True).name()
+        return self.block._var_recursive(attr_var_name)
+
+    def _vars_attr(self, name):
+        """
+        Get the Variables attribute  by name.
```
Comment (chenwhql) [inline python/paddle/fluid/framework.py:3301]: 同上

**Review by chenwhql — APPROVED** [review]
LGTM

File: python/paddle/fluid/framework.py (hunk 2 above)
Comment (Aurelius84) [reply python/paddle/fluid/framework.py:3301]: 我后面删除下

**Pull request merged**